pkgs/development/libraries/dpp/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libopus
+, openssl
+, zlib
+, libsodium
+, gcc
+, pkg-config
+, autoPatchelfHook
+, lib
+}:
+let
+   libExt = if stdenv.hostPlatform.isDarwin then ".dylib" else ".so";
+in
```

Comment:
```suggestion

```
we already have that in stdenv.hostPlatform.extensions.sharedLibrary and I also moved it closer to the usage

Reply:
thx, I moved it closer to the usage as requested